lib/longtail/stackoverflow/posts.pl

```diff
@@ -61,7 +61,31 @@
     }
 
     print qq(\nTYPE: $name\t$data->{src_domain}\n);
-    
+
+    my %post_links = ();
+    if (-e "$stack_dir/$data->{src_domain}/PostLinks.xml") {
```

Comment:
If this doesn't exist, don't we want do more than skip it?  We should probably be better about `die`ing if files don't exist that we expect.